radar-auth/src/main/java/org/radarbase/auth/token/DataRadarToken.kt

```diff
@@ -12,7 +12,7 @@ data class DataRadarToken(
      * Get all roles defined in this token.
      * @return non-null set describing the roles defined in this token.
      */
-    override val roles: Set<AuthorityReference>,
+    override val roles: Set<AuthorityReference>?,
```

Comment:
Ideally this is not null for all authentication methods. If a person has no roles, this can be an empty list.